ChangeLog.txt

```diff
@@ -5,12 +5,15 @@ This release is save-game compatible with previous versions
 
 MAJOR:
 - Added a new government type: Legion government type for the legions, based upon the government in BTWK (credits to the BTWK team)
+- Extensive overhaul and improvement of province/title flags
 
 MINOR:
-- Modified the culture and religion of the legion leader. The legion leader has now the culture and religion of the restaurer instead the roman culture.
+- Modified the culture and religion of the leaders of restored Legions. The legion leader has now the culture and religion of the restorer instead of Roman culture.
 - Fixed the issue with the Italian Prefecture having 20 000 in dignity rather than 200, which caused it to stay the primary title even though the holders held empires or kingdoms
 - Modified the Sassanid government type to prevent Zoroastrian characters outside the Sassanids from receiving it
 - Solved the issue which caused temple vassals of the Sassanids to have the wrong holding type
+- Certain Gnostic heresies now use a special flavor image for heretic events rising
```

Comment:
"Certain Gnostic heresies" refer to certain Christian heresies that are Gnostic, not heresies of a Gnostic group.